app/models/api/submission.rb

```diff
@@ -1,6 +1,5 @@
 module API
   class Submission < Base
-    has_many :entries, class_name: 'SubmissionEntry'
```

Comment:
This is going because we're no longer returning it from the API, right?

Reply:
it's going because we don't want to the frontend to be able to pull all the entries. it should be able to function without them, and allowing it opens up some pretty horrible performance issues for submissions that may have tens of thousands of entries.

Reply:
Great, thanks